interface/controller/definition_controller_test.go

```diff
@@ -0,0 +1,124 @@
+package controller
+
+import (
+	"encoding/json"
+	"io/ioutil"
+	"net/http"
+	"net/http/httptest"
+	"testing"
+
+	"github.com/OrlandoRomo/academy-go-q32021/domain/model"
+	"github.com/gorilla/mux"
+	"github.com/stretchr/testify/assert"
+	"github.com/stretchr/testify/mock"
+)
+
+type MockInteractor struct {
+	mock.Mock
+}
+
+func (m MockInteractor) Get(term string) (*model.List, error) {
+	args := m.Called(term)
+	return args.Get(0).(*model.List), args.Error(1)
+}
+func (m MockInteractor) GetFromCSV(id string) (*model.List, error) {
+	args := m.Called(id)
+	return args.Get(0).(*model.List), args.Error(1)
+}
+func (m MockInteractor) GetConcurrent() (*model.List, error) {
+	args := m.Called()
+	return args.Get(0).(*model.List), args.Error(1)
+}
+
+func TestGetDefinitions(t *testing.T) {
+	mockInteractor := MockInteractor{}
+	t.Run("success - valid request", func(t *testing.T) {
```

Comment:
Be consistent with how you layout the tests througout the app, whether that's with the iterative format or with this individual tests format, I would suggest using the iterative format, it's easier to read